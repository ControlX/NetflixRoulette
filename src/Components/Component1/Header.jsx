import React from 'react'

export default function Header(){
    return (
        <p className='parent-header-primary-font-properties'><strong>netflix</strong>roulette</p>
    )
}