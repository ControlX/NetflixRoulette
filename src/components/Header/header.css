.header-parent-label-properties {
    color: #F65261;
    font-size: 24px;
    margin-left: 5%;
    position: relative;
}

.header-btn-add-movie-properties{
    color: #F65261;
    margin-right: 5%;
    float: right;
    position: absolute;
    right: 0;
    top: 0;
    background-color: #55555563;
    font-size: 20px;
    padding: 15px 25px;
    margin-top: 1%;
}

.header-lbl-find-properties{
    color: #FFFFFF;
    font-size: 38px;
    margin-left: 10%;
}

.header-input-properties{
    position: relative;
    margin-left: 10%;
    margin-top: 3%;
    width: 400px;
    height: 50px;
    background-color: rgba(0, 0, 0, 0.226);
    padding-left: 20px;
    font-size: 20px;
    color: #FFFFFF;
}

.header-btn-search-properties{
    background-color: #F65261;
    color: white;
    height: 55px;
    padding-left: 60px;
    padding-right: 60px;
    margin-left: 2%;
    font-size: 18px;
}