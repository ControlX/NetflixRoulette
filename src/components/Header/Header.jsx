import React from 'react';
import './header.css';

export default function Header(props){
    return (
        <div>
        <p className='header-parent-label-properties'><strong>netflix</strong>roulette</p>
        <button className='header-btn-add-movie-properties' onClick={()=>props.onAddAction()}>+ ADD MOVIE</button>
        <br /><br />

        <label className='header-lbl-find-properties'>FIND YOUR MOVIE</label>
        <br />
        <input className='header-input-properties' placeholder={'What do you want to watch?'}/>
        <button className='header-btn-search-properties'>SEARCH</button>
        </div>
    )
}