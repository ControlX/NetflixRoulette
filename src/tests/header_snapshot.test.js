import React from 'react';
import Header  from '../components/Header';
import renderer from 'react-test-renderer';

test("===Header Snapshot Test===", () => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
})