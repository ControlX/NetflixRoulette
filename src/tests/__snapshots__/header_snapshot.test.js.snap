// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`===Header Snapshot Test=== 1`] = `
<div>
  <p
    className="header-parent-label-properties"
  >
    <strong>
      netflix
    </strong>
    roulette
  </p>
  <button
    className="header-btn-add-movie-properties"
    onClick={[Function]}
  >
    + ADD MOVIE
  </button>
  <br />
  <br />
  <label
    className="header-lbl-find-properties"
  >
    FIND YOUR MOVIE
  </label>
  <br />
  <input
    className="header-input-properties"
    onChange={[Function]}
    placeholder="What do you want to watch?"
  />
  <button
    className="header-btn-search-properties"
    onClick={[Function]}
  >
    SEARCH
  </button>
</div>
`;
